syncopy/spikes/psth.py

```diff
@@ -3,10 +3,11 @@
 
 
 def psth(trl_dat,
-         cu_combos,
-         trl_start=0,
-         onset=0,
-         bins=None,
+         trl_start,
+         onset,
+         trl_end,
+         chan_unit_combs=None,
+         tbins=None,
          samplerate=1000):
```

Comment:
not part of this changeset, but the docstring for the `samplerate` parameter is missing